simple/tests/stats/test_data/stat_var_hierarchy_generator/expected/three_unrelated_svs_svgs.json

```diff
@@ -0,0 +1,136 @@
+[
+ {
+  "svg_id": "c/g/Person_Gender-Female_Race-Asian",
+  "svg_name": "Person With Gender = Female, Race = Asian",
+  "sv_ids": [
+   "sv1"
+  ],
+  "parent_svg_ids": [
+   "c/g/Person_Gender_Race-Asian",
+   "c/g/Person_Gender-Female_Race"
+  ],
+  "child_svg_ids": []
+ },
+ {
+  "svg_id": "c/g/Coal_EnergySource-CokeCoal",
+  "svg_name": "Coal With EnergySource = CokeCoal",
+  "sv_ids": [
+   "sv2"
+  ],
+  "parent_svg_ids": [
+   "c/g/Coal_EnergySource"
+  ],
+  "child_svg_ids": []
+ },
+ {
```

Comment:
TODO:  Might be worth doing some pruning... we probably should ignore Thing.

Reply:
Added TODO. Let me also understand what kinds of pruning we can do and do it in a dedicated PR.